package.json

```diff
@@ -71,7 +71,8 @@
         "onLanguage:javascript",
         "onLanguage:python",
         "onLanguage:csharp",
-        "onLanguage:yaml"
+        "onLanguage:yaml",
+        "onCommand:aws.lambda.invokeForm"
```

Comment:
We should rename these symbols now, it will be difficult if we wait until later. The primary purpose of this new UI is to be a UI for launch configs. So this symbol could be named `aws.launchConfigForm`. 

Similar comment for all other related "invoke lambda" symbols.